react/features/invite/components/add-people-dialog/web/InviteContactsForm.js

```diff
@@ -0,0 +1,495 @@
+                ],
+                content: t('addPeople.telephone', { number: number.number }),
+                description,
+                isDisabled: !number.allowed,
+                elemBefore: telephoneIcon,
+                item: number,
+                tag: {
+                    elemBefore: telephoneIcon
+                },
+                value: number.number
+            };
+        });
+
+        return [
+            ...userDisplayItems,
+            ...numberDisplayItems
+        ];
+    }
+
+    _query: (string) => Promise<Array<Object>>;
+
+    _renderFooterText: () => Object;
+
+    /**
+     * Sets up the rendering of the footer text, if enabled.
+     *
+     * @returns {Object | undefined}
+     */
+    _renderFooterText() {
+        const { _footerTextEnabled, t } = this.props;
+        let footerText;
+
+        if (_footerTextEnabled) {
+            footerText = {
+                content: <div className = 'footer-text-wrap'>
+                    <div>
+                        <span className = 'footer-telephone-icon'>
+                            <Icon src = { IconPhone } />
+                        </span>
+                    </div>
+                    { translateToHTML(t, 'addPeople.footerText') }
+                </div>
+            };
+        }
+
+        return footerText;
+    }
+
+    _onClearItems: () => void;
+
+    /**
+     * Clears the selected items from state and form.
+     *
+     * @returns {void}
+     */
+    _onClearItems() {
+        if (this._multiselect) {
+            this._multiselect.setSelectedItems([]);
+        }
+        this.setState({ inviteItems: [] });
+    }
+
+    /**
+     * Renders the add/cancel actions for the form.
+     *
+     * @returns {ReactElement|null}
+     */
+    _renderFormActions() {
+        const { inviteItems } = this.state;
+        const { t } = this.props;
+
+        if (!inviteItems.length) {
+            return null;
+        }
+
+        return (
+            <div className = 'invite-more-dialog invite-buttons'>
+                <a
+                    className = 'invite-more-dialog invite-buttons-cancel'
+                    onClick = { this._onClearItems }>
+                    {t('dialog.Cancel')}
+                </a>
+                <a
+                    className = 'invite-more-dialog invite-buttons-add'
+                    onClick = { this._onSubmit }>
+                    {t('addPeople.add')}
+                </a>
+            </div>
+        );
+    }
+
+    /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @private
+     * @returns {ReactElement|null}
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
+        }
+
+        const { t } = this.props;
+        const supportString = t('inlineDialogFailure.supportMsg');
+        const supportLink = interfaceConfig.SUPPORT_URL;
+        const supportLinkContent = supportLink && (
```

Comment:
here you'd still be rendering the inline message below, I think we'd need to skip that too.

Reply:
Oh damn, you're right